src/themes.js

```diff
@@ -1,82 +1,76 @@
-import { FONTS_AVAILABLE } from './constants'
+import { FONTS_AVAILABLE } from './constants';
 
-export default http => ({
-  list: args => getThemes(http, args),
-  get: args => getTheme(http, args),
-  create: args => createTheme(http, args),
-  delete: args => deleteTheme(http, args),
-  update: args => updateTheme(http, args)
-})
+export default http => new Themes(http);
 
-const getThemes = (http, { page, pageSize } = {}) => {
-  return http.request({
-    method: 'get',
-    url: '/themes',
-    params: {
-      page,
-      page_size: pageSize
+class Themes {
+  constructor(_http) {
+    this._http = _http;
+  }
+
+  create({ background, colors, font, hasTransparentButton, name }  = {}) {
+    // check if required properties are defined
+    if ([name, font, colors].includes(undefined)) {
+      throw new Error(`Please add the required fields`);
     }
-  })
-}
 
-const getTheme = (http, { id }) => {
-  return http.request({
-    method: 'get',
-    url: `/themes/${id}`
-  })
-}
+    if (!FONTS_AVAILABLE.includes(font)) {
+      throw new Error(`It's not a valid font`);
```

Comment:
I would add the name of the font here to make the error more specific and clearer for the developer.

```js
`${font} is not a valid font`
```